### ISSUE TITLE
Remove and reinstall?

### ISSUE THREAD

**Issue by anthrox** [MicrosoftDocs/OfficeDocs-DeployOffice#63]
if i already have a office 365 click to run installed and then a user want Visio volume licence after do i have to uninstall office and then reinstall it with the Visio flags?

---
#### Document details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 1e6c0641-503d-7959-2afb-839821cc792f
* Version Independent ID: 046c52d9-4fbe-4fe7-0778-5c1d4138f69e
* Content: [Use the Office Deployment Tool to install volume licensed editions of Visio 2016 and Project 2016](https://docs.microsoft.com/en-au/DeployOffice/use-the-office-deployment-tool-to-install-volume-licensed-editions-of-visio-2016?redirectSourcePath=%252fen-us%252farticle%252f82691bd7-a3d5-47ca-8d8a-0ee43ec2c01f)
* Content Source: [DeployOffice/use-the-office-deployment-tool-to-install-volume-licensed-editions-of-visio-2016.md](https://github.com/MicrosoftDocs/OfficeDocs-DeployOffice/blob/live/DeployOffice/use-the-office-deployment-tool-to-install-volume-licensed-editions-of-visio-2016.md)
* Service: **o365-proplus-itpro**
* GitHub Login: @DHB-MSFT
* Microsoft Alias: **danbrown**

**Comment by DHB-MSFT** [comment]
No, you don't have to uninstall the existing version of Office before installing Visio.

**Comment by Brada-srb** [comment]
This is not correct, spend an hour with Microsoft Support, and at the end - need to de install office 365 Bussines premium
